src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.props

```diff
@@ -48,10 +48,10 @@ Copyright (c) .NET Foundation. All rights reserved.
   <ItemGroup Condition=" '$(DisableImplicitFrameworkReferences)' != 'true' and '$(TargetFrameworkIdentifier)' == '.NETCoreApp'">
     <PackageReference Include="Microsoft.NETCore.App" Version="$(RuntimeFrameworkVersion)" IsImplicitlyDefined="true" />
 
-    <!-- For libraries targeting .NET Core 2.0 or higher, don't include a dependency on Microsoft.NETCore.App in the package produced by pack.
-         Packing an app (for example a .NET CLI tool) should include the Microsoft.NETCore.App package dependency. -->
+    <!-- For targeting .NET Core 2.0 or higher, don't include a dependency on Microsoft.NETCore.App in the package produced by pack.
+         Packing an DotnetCliTool should include the Microsoft.NETCore.App package dependency. -->
     <PackageReference Update="Microsoft.NETCore.App"
-                      Condition="('$(OutputType)' != 'Exe') And ('$(_TargetFrameworkVersionWithoutV)' != '') And ('$(_TargetFrameworkVersionWithoutV)' >= '2.0')"
+                      Condition="('$(_TargetFrameworkVersionWithoutV)' != '') And ('$(_TargetFrameworkVersionWithoutV)' >= '2.0') And ('$(PackageType)' != 'DotnetCliTool')"
```

Comment:
Do we want another escape hatch than setting PackageType? What else does that impact?

Reply:
I would say we already have another escape hatch: put the following in your project:

```xml
<ItemGroup>
    <PackageReference Update="Microsoft.NETCore.App" PrivateAssets="None" />
</ItemGroup>
```

Reply:
Good point. I'm convinced. :)